redhat-cop/casl-ansible
# Fix GCP Registry Storage

#### What does this PR do?
This PR updates CASL to utilize the openshift-ansible OSv3 variables to utilize a GCP cloud bucket as the backend storage for the registry. This allows us to remove some of the custom logic that we had built in (which were causing some issues) and now provides a working backend for the registry.

#### How should this be manually tested?
Create a 1x1x1 cluster and kick off a build. You will now see that the image is successfully pushed to the registry. Previously we would see a 500/403 error reported showing that we didn't have permissions to utilize the GCP storage bucket.

#### Is there a relevant Issue open for this?
N/A

#### Who would you like to review this?
cc: @redhat-cop/casl
